server/resp/src/main/java/org/infinispan/server/resp/commands/connection/SELECT.java

```diff
@@ -24,8 +29,16 @@ public SELECT() {
    public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
                                                       ChannelHandlerContext ctx,
                                                       List<byte[]> arguments) {
-      ByteBufferUtils
-            .stringToByteBufAscii("-ERR Select not supported in cluster mode\r\n", handler.allocator());
+      String db = new String(arguments.get(0), StandardCharsets.US_ASCII);
+      Cache<byte[], byte[]> cache = handler.respServer().getCacheManager().getCache(db);
+      if (cache != null) {
+         ConnectionMetadata metadata = ConnectionMetadata.getInstance(ctx.channel());
+         handler.setCache(cache.getAdvancedCache().withSubject(metadata.subject()).withMediaType(MediaType.APPLICATION_OCTET_STREAM, MediaType.APPLICATION_OCTET_STREAM));
+         metadata.cache(cache.getName());
+         Consumers.OK_BICONSUMER.accept(null, handler.allocator());
+      } else {
+         ByteBufferUtils.stringToByteBuf("-ERR DB index is out of range\r\n", handler.allocator());
+      }
```

Comment:
My Redis/RESP knowledge is lacking here. I understand that we're adding an alias of `0` by default, but what must be configured in order for an Infinispan user to do `SELECT 1`? This should be documented.

Reply:
I've added a chapter to the redis connector docs